Softeq.XToolkit.Permissions.iOS/Softeq.XToolkit.Permissions.iOS.csproj

```diff
@@ -30,6 +29,8 @@
         </MtouchVerbosity>
         <MtouchLink>
         </MtouchLink>
+        <LangVersion>8.0</LangVersion>
+        <Nullable>enable</Nullable>

```

Comment:
Please set lang version and `<Nullable>` in the shared `<PropertyGroup>`, instead of build configurations (VS default behavior)